manager/integration/tests/common.py

```diff
@@ -1057,8 +1057,7 @@ def clients(request):
     clis = get_clients(hosts)
 
     def finalizer():
-        client = next(iter(clis.values()))
-        cleanup_client(client)
+        cleanup_client()
 
     request.addfinalizer(finalizer)
 
```

Comment:
Do we still need the `client = next(iter(clis.values()))` below?

Reply:
No, I've removed it.